search/utm_mgrs_coords_match.cpp

```diff
@@ -0,0 +1,251 @@
+#include "search/utm_mgrs_coords_match.hpp"
+
+#include "base/string_utils.hpp"
+
+#include "platform/utm_mgrs_utils.hpp"
+
+#include <iterator>
+#include <string>
+#include <utility>
+#include <iostream>
+#include <math.h>
+
+using namespace std;
+
+namespace search
+{
+
+bool IsSpace(char ch);
+string::size_type FindNonSpace(string const & txt, string::size_type startPos);
```

Comment:
Well, string::size_type is a useless overkill (but with correct semantics). Use size_t instead.

Reply:
Replaced with `size_t`.